### ISSUE TITLE
Forecasting for categorical target parameter in DataSetTimeSeries?

### ISSUE THREAD

**Issue by DominikKrock** [jdb78/pytorch-forecasting#91]
Hello @jdb78,

Is there the ability to use the forcasting package to predict categorical timeseries? I would like to use the package for a multiclass classification problem. So, in my problem the target parameter in DataSetTimeSeries is categorical and not numeric.  

If this is not supported right now can you estimate whether it is possible to integrate this feature and where in the code do you see the biggest adjustments/changes which have to be made?

**Comment by jdb78** [comment]
It is not implemented natively in the dataset but it should be fairly simple to get there (guess 10 or less lines). First, encode your target as integer. Then, create a new metric, e.g. you could directly use https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html with no reduction.

**Comment by jdb78** [comment]
It does work now but the prediction plots are still rather meaningless. Feel welcome to improve this functionality.

**Comment by jsolson4** [comment]
@DominikKrock did you implement metrics in addition to this loss function? I have a loss function working with the stock regression-based metrics, but classification-based metrics are all throwing errors. I am working on understanding what I need to do to capture proper metrics.
